CIP-????/README.md

```diff
@@ -0,0 +1,113 @@
+---
+CIP: ?
+Title: Deterministic universal almost-unique Plutus Constructors
+Category: Plutus
+Status: Proposed
+Authors:
+    - Niels Mündler <niels@opshin.dev>
+Implementors: [Niels Mündler <niels@opshin.dev>]
```

Comment:
```suggestion
Implementors:
    - Niels Mündler <niels@opshin.dev>
```
What you submitted may be correct YAML but CIP-0001 indicates 1 entry per line (for consistency, readability & parseability).

But @nielstron I wouldn't commit this yet since I don't think our current process can list your as an implementor.  I would happily stand corrected on this (@KtorZ @michaelpj)... especially since I don't work directly with Plutus... but regardless of its implementation in something like OpShin, wouldn't this only be "Implemented" when added to Plutus Core (something you can't do yourself)?

Reply:
This proposal does not require any changes or implementation to Plutus Core. It is more of a statement on how to handle situations that arise when dealing with different implementations or usages of the same contract across different languages.

Reply:
excellent @nielstron, thanks 😎